<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8" />
	<title>eggNOG-mapper</title>
	<meta name="description" content="Latest updates and statistic charts">
	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, shrink-to-fit=no">

	
	<!-- template -->
    
	<link href="static/metronic_src/assets/vendor/base/vendors.bundle.css" rel="stylesheet" type="text/css" />
	<link href="static/metronic_src/assets/demo/default/base/style.bundle.css" rel="stylesheet" type="text/css" />

	<script src="static/metronic_src/assets/demo/default/base/scripts.bundle.js"></script>
	<script src="static/metronic_src/assets/vendor/base/vendors.bundle.js"></script>
	
	<!--begin::Web font -->
	<script src="static/metronic_src/webfont.js"></script>
	<script>
		WebFont.load({
			google: { "families": ["Poppins:300,400,500,600,700", "Roboto:300,400,500,600,700"] },
			active: function () {
				sessionStorage.fonts = true;
			}
		});
	</script>
	<!--end::Web font -->
	
	<!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-783841-20"></script>
	<script>
	  window.dataLayer = window.dataLayer || [];
	  function gtag(){dataLayer.push(arguments);}
	  gtag('js', new Date());
	  
	  gtag('config', 'UA-783841-20');
	  console.log("done");
	</script>
	<!-- End::Google analytics -->

	<script src="static/metronic_src/vue.min.js"></script>
	<script src="static/metronic_src/axios.min.js"></script>

	<link rel="shortcut icon" href="static/metronic_src/assets/demo/default/media/img/logo/favicon.ico" />

	<script src="static/metronic_src/vue-tables-2.min.js"></script>
	
	<script>
	  Vue.use(VueTables.ClientTable, [theme = 'bootstrap3']);
	</script>

	<style>
	  .table-emapper td {
	  height: 100px;
	  }
	  .table-emapper td > .scrollable {
	  height: 100%;
	  overflow: auto;
	  }
	  .table-emapper thead td {
	  white-space: nowrap;
	  }
	  .nowrap {
	  white-space: nowrap;
	  }
	</style>
</head>

<body
  class="m-page--fluid m--skin- m-content--skin-light2 m-header--fixed m-header--fixed-mobile m-aside-left--enabled m-aside-left--skin-dark m-aside-left--fixed m-aside-left--offcanvas m-footer--push m-aside--offcanvas-default">
        <!-- Begin::Vue App -->
        
        <div id="emapper_app">
            <!-- begin::Page -->
            <div class="m-grid m-grid--hor m-grid--root m-page">
                    <div id='header_app'>
                            <header id="m_header" class="m-grid__item    m-header " m-minimize-offset="200" m-minimize-mobile-offset="200">
<div class="m-container m-container--fluid m-container--full-height">
    <div class="m-stack m-stack--ver m-stack--desktop">

        <!-- BEGIN: Brand -->
        <div class="m-stack__item m-brand  m-brand--skin-dark ">
            <div class="m-stack m-stack--ver m-stack--general">
                <div class="m-stack__item m-stack__item--middle m-brand__logo">
                    <a href="/" class="m-brand__logo-wrapper">
                      <!-- <span  style="font-size:140%;color:white;">Comp</span> -->
                                    

                      <span  style="font-size:140%;color:indianRed;">GenomicsTools</span>
                        <!-- <img alt="" src="static/assets/demo/default/media/img/logo/logo.png" /> -->
                    </a>
                </div>
                <div class="m-stack__item m-stack__item--middle m-brand__tools">

                    <!-- BEGIN: Left Aside Minimize Toggle -->
                    <a href="javascript:;" id="m_aside_left_minimize_toggle"
                        class="m-brand__icon m-brand__toggler m-brand__toggler--left m--visible-desktop-inline-block  ">
                        <span></span>
                    </a>

                    <!-- END -->

                    <!-- BEGIN: Responsive Aside Left Menu Toggler -->
                    <a href="javascript:;" id="m_aside_left_offcanvas_toggle"
                        class="m-brand__icon m-brand__toggler m-brand__toggler--left m--visible-tablet-and-mobile-inline-block">
                        <span></span>
                    </a>

                    <!-- END -->

                    <!-- BEGIN: Responsive Header Menu Toggler -->
                    <a id="m_aside_header_menu_mobile_toggle" href="javascript:;"
                        class="m-brand__icon m-brand__toggler m--visible-tablet-and-mobile-inline-block">
                        <span></span>
                    </a>

                    <!-- END -->

                    <!-- BEGIN: Topbar Toggler -->
                    <a id="m_aside_header_topbar_mobile_toggle" href="javascript:;"
                        class="m-brand__icon m--visible-tablet-and-mobile-inline-block">
                        <i class="flaticon-more"></i>
                    </a>
                    <!-- BEGIN: Topbar Toggler -->
                </div>
            </div>
        </div>

        <!-- END: Brand -->
        <div class="m-stack__item m-stack__item--fluid m-header-head" id="m_header_nav">
                
          <img src="static/metronic_src/eggnog-mapper-logo.jpg" style="padding:1%;">
        
          <!-- ACTIVE TOOL? -->

        </div>
    </div>
</div>
</header>
            </div>
<!-- begin::AppBody -->
<div class="m-grid__item m-grid__item--fluid m-grid m-grid--ver-desktop m-grid--desktop m-body">

    <button class="m-aside-left-close  m-aside-left-close--skin-dark " id="m_aside_left_close_btn"><i
class="la la-close"></i></button>
<div id="m_aside_left" class="m-grid__item	m-aside-left  m-aside-left--skin-dark ">

<div id="m_ver_menu" class="m-aside-menu  m-aside-menu--skin-dark m-aside-menu--submenu-skin-dark "
m-menu-vertical="1" m-menu-scrollable="1" m-menu-dropdown-timeout="200" style="position: relative;">
<ul class="m-menu__nav  m-menu__nav--dropdown-submenu-arrow ">

<li class="m-menu__section ">
<h4 class="m-menu__section-text">Tools</h4>
<i class="m-menu__section-icon flaticon-more-v2"></i>
</li>
<li class="m-menu__item  m-menu__item--submenu" aria-haspopup="true" m-menu-submenu-toggle="hover">
<a href="/" class="m-menu__link m-menu__toggle">
<i class="m-menu__link-icon flaticon-layers"></i>
<span class="m-menu__link-text">eggNOG-mapper</span>
<i class="m-menu__ver-arrow la "></i>
</a>
</li>
<li class="m-menu__section ">
<h4 class="m-menu__section-text">Support</h4>
<i class="m-menu__section-icon flaticon-more-v2"></i>
</li>
<li class="m-menu__item  m-menu__item--submenu" aria-haspopup="true" m-menu-submenu-toggle="hover">
<a href="https://github.com/eggnogdb/eggnog-mapper/wiki/FAQ---Frequently-Asked-Questions" target='_blank'
class="m-menu__link m-menu__toggle">
<i class="m-menu__link-icon flaticon-interface-9"></i>
<span class="m-menu__link-text">Frequent Questions</span>
<i class="m-menu__ver-arrow la"></i>
</a>
</li>
<li class="m-menu__item  m-menu__item--submenu" aria-haspopup="true" m-menu-submenu-toggle="hover">
<a href="https://github.com/eggnogdb/eggnog-mapper/wiki/eggNOG-mapper-v2" target='_blank'
class="m-menu__link m-menu__toggle">
<i class="m-menu__link-icon flaticon-interface-9"></i>
<span class="m-menu__link-text">Documentation</span>
<i class="m-menu__ver-arrow la"></i>
</a>
</li>
<li class="m-menu__item  m-menu__item--submenu" aria-haspopup="true" m-menu-submenu-toggle="hover">
<a href="https://github.com/eggnogdb/eggnog-mapper" target='_blank' class="m-menu__link m-menu__toggle">
<i class="m-menu__link-icon flaticon-interface-9"></i><span class="m-menu__link-text">Standalone
version</span>
<i class="m-menu__ver-arrow la"></i>
</li>

<li class="m-menu__item  m-menu__item--submenu" aria-haspopup="true" m-menu-submenu-toggle="hover">
<a href="/about" target='_blank' class="m-menu__link m-menu__toggle">
<i class="m-menu__link-icon flaticon-interface-9"></i><span class="m-menu__link-text">About</span>
<i class="m-menu__ver-arrow la"></i>
</a>
</li>


</ul>
</div>
<div class="ps__rail-x" style="left: 0px; bottom: 0px;">
<div class="ps__thumb-x" tabindex="0" style="left: 0px; width: 0px;"></div>
</div>
<div class="ps__rail-y" style="top: 0px; height: 876px; right: 4px;">
<div class="ps__thumb-y" tabindex="0" style="top: 0px; height: 643px;"></div>
</div>
</div>

<!-- begin::Main -->
<div class="m-grid__item m-grid__item--fluid m-wrapper">
    <div class="m-content">
      <div class="row">
        <div class='col-xl-6'>
          <div class="m-portlet">
  
      <div class="m-portlet__head">
          <div class="m-portlet__head-caption">
              <div class="m-portlet__head-title">
  
                  <span class="m-portlet__head-icon m--hide">
                      <i class="la la-gear"></i>
                  </span>
                  <h3>
                      Annotate FASTA file
                  </h3>
  
              </div>
          </div>
      </div>


<!--begin::Form-->
<form class="m-form" action='/submit_job' method='post' enctype="multipart/form-data">

    <input type="hidden" name="csrfmiddlewaretoken" value="UCKEl404brIzBSvM6wv5j8DQ5FRIo3RFWKPcTTzNvikE7yAk04MeWGmJar8PR0E6">
      <div class="m-portlet__body">
          <div class="m-form__section m-form__section--first">
              <div class="form-group m-form__group">
                  <label>Upload Sequences</label>
                  <span class="m-form__help">(Up to 100,000 CDS/protein sequences in <b>FASTA</b> format)</span>
          <!-- <span class="m-form__help">(Up to 100,000 protein sequences in <b>FASTA</b> format)</span> -->
                  <input type='file' name='param_fastafile' class='form-control m-input' required>
                  <br>
                  <label>Email address:</label>
                  <span class="m-form__help">(Required for job scheduling and notifications)</span>
                  <input type="email" class="form-control m-input" placeholder="Enter email" name='param_email'
                      v-model='params.param_email' pattern=".+@.+\..+" required>


                  <input type='hidden' name='param_search_mode' value='diamond'>
                  <input type='hidden' name='param_database' value='none'>
                  <br>
                  <!-- <label>Sequence mapping mode</label> -->
                  <!-- <select class="form-control" name='param_search_mode' v-model='params.param_search_mode'> -->
                  <!--     <option value="diamond" selected>DIAMOND search </option> -->
                  <!--     <option value="hmmer">HMM search </option> -->
                  <!-- </select> -->

                  <label>Taxonomic Scope</label>
                  <span class="m-form__help">(annotations will only be transferred from orthologs in the selected taxa
                      group)</span>

                  <select class="form-control" name='param_tax_scope' v-model='params.param_tax_scope'>
                      <option value="auto">Auto adjust per query (RECOMMENDED)</option>
                      
                      <option value="2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Bacteria - 2</option>
                      
                      <option value="1224">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Proteobacteria - 1224</option>

                </select>
                <br>
                <label>Orthology restrictions</label>
                <div>
                    <select class="form-control" name='param_otype' v-model='params.param_otype'>
                        <option value="all">Transfer annotations from any ortholog</option>
                        <option value="one2one">Transfer annotations from <b>one-to-one orthology only</b>
                        </option>
                    </select>
                </div>

                <br>
            </div>


            <div class="kt-portlet__body">
                <!--begin::Accordion-->

                <div class="accordion accordion-solid accordion-toggle-plus" id="accordionExample6">
                    <div class="card">
                        <div class="card-header" id="headingOne6">
                            <div class="card-title collapsed" data-toggle="collapse" data-target="#collapseOne6"
                                aria-expanded="false" aria-controls="collapseOne6">
                                <i class="flaticon-pie-chart-1"></i> Advanced Options
                            </div>
                        </div>
                        <div id="collapseOne6" class="collapse" aria-labelledby="headingOne6"
                            data-parent="#accordionExample6" style="">
                            <div class="card-body">
                                <label>Gene Ontology evidence</label>
                                <div>
                                    <select class="form-control" name='param_goevidence'
                                        v-model='params.param_goevidence'>
                                        <option value="non-electronic">Transfer non-electronic annotations </option>
                                        <option value="experimental">Transfer annotations with experimental evidence
                                            only</b>
                                        </option>
                                    </select>
                                </div>




                                <label>Seed ortholog detection </label>
                                <div>
                                    <label>Minimum hit e-value</label>
                                    <input class="form-control" type="number" type='text' min=0.0 max=1.0
                                        step="0.001" name='param_seed_evalue' value='0.001'>
                                    <label>Minimum hit bit-score</label>
                                    <input class="form-control" type="number" type='text' name='param_seed_score'
                                        value='60' type='text' min=0 max=999999>
                                    <label>Minimum % of query coverage</label>
                                    <input class="form-control" type="number" type='text' min=0 max=100
                                        name='param_query_cov' value=20>
                                    <label>Minimum % of subject coverage</label>
                                    <input class="form-control" type="number" type='text' min=0 max=100
                                        name='param_subject_cov' value=0>

                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                </div>
                </div>
                </div>
                <!--end::Accordion-->

                <div class="m-portlet__foot m-portlet__foot--fit">

                <div class="m-form__actions m-form__actions">
                    <button type="submit" class="btn btn-primary">Submit</button>
                    <button type="reset" class="btn btn-secondary">Cancel</button>
                </div>
            </div>
    </form>
    <!--end::Form-->

    <div class="m-portlet__body">
        <p>
            <h3>Citation:</h3> <br>
            <b>Fast genome-wide functional annotation through orthology assignment by
                eggNOG-mapper.</b> <br>
            <i>Jaime Huerta-Cepas, Kristoffer Forslund, Luis Pedro Coelho, Damian Szklarczyk,
                Lars
                Juhl Jensen,
                Christian von Mering & Peer Bork</i>.<br>
            <a target="_blank" href="https://doi.org/10.1093/molbev/msx148" style="color:royalblue;">Mol Biol Evol
                (2017)
                doi: 10.1093/molbev/msx148. </a>
            <br><br>


            <b>eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated
                orthology resource based on 5090 organisms and 2502 viruses.</b> <br>
            <i>Jaime Huerta-Cepas, Damian Szklarczyk, Davide Heller, Ana Hernández-Plaza, Sofia
                K Forslund, Helen Cook, Daniel R Mende, Ivica Letunic, Thomas Rattei, Lars
                J Jensen, Christian von Mering, Peer Bork</i><br>
            <a target="_blank" href="https://doi.org/10.1093/nar/gky1085"> Nucleic Acids Res. 2019 Jan 8;
                47(Database issue): D309–D314. doi: 10.1093/nar/gky1085 </a>

        </p>
    </div>
</div>

      </div>
      <div class='col-xl-4'>
        <div class="m-portlet">
  <div class="m-portlet__head">
    <div class="m-portlet__head-caption">
      <div class="m-portlet__head-title">
	<span class="m-portlet__head-icon m--hide">
	  <i class="la la-gear"></i>
	</span> 
	<h3>
	  Method Overview
	</h3>
      </div>
    </div>
  </div>
  <div class="m-portlet__body" style="text-align:center;">
	<p style="text-align:justify;">
		<span><b>eggNOG-mapper v2</b></span> is a tool for functional annotation of large sets
		of
		sequences
		based on
		fast orthology assignments using precomputed <b>eggNOG v5.0 </b> clusters and phylogenies. The
		method is illustrated in the following
		figure:
	</p>
	<a src='/static/emapper_workflow.png'>
	  <img src='/static/emapper_workflow.png' style='width:66%;'>
	</a>
  </div>
</div>

      </div>
    </div>
    <div class="row">
        <!-- Server status component -->
        <server-status></server-status>
    </div>
  </div>
</div>
<!-- End::Main -->


			</div>
			<!-- end::AppBody -->
                </div>
                

                <!-- end::Page -->
        </div>
        <!-- End::Vue App -->
</body>
<!-- Server-status data for Vue components -->
<script>
    var jobslist = [{'email': '***@gmail.com', 'jobname': 'MM_eg8_4ckd', 'numseqs': '4924', 'databases': 'none', 'status': 'Running', 'submitted': '2020-03-14', 'elapsed': '32:44', 'original_file': 'Galaxy5-[Prokka_on_data_1__faa] (2).fasta', 'tax_scope': '1236'}, {'email': '***@163.com', 'jobname': 'MM_zdollzdx', 'numseqs': '626', 'databases': 'none', 'status': 'Running', 'submitted': '2020-03-14', 'elapsed': '42:01', 'original_file': 'output13020200314112857.pep.fa', 'tax_scope': 'auto'}, {'email': '***@qq.com', 'jobname': 'MM_04hiedtv', 'numseqs': '6646', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '00:54:39', 'original_file': '纯蛋白序列.fasta', 'tax_scope': 'auto'}, {'email': '***@qq.com', 'jobname': 'MM_ppm6m8um', 'numseqs': '10182', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '01:19:25', 'original_file': 'Ea588_PacBio_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@live.unc.edu', 'jobname': 'MM_3t1ihcqd', 'numseqs': '3296', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '01:04:58', 'original_file': 'GCF_000766795.1_Polaribacter_sp._Hel1_85_protein.faa', 'tax_scope': 'auto'}, {'email': '***@live.unc.edu', 'jobname': 'MM_aiw0caj7', 'numseqs': '3344', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '01:08:36', 'original_file': 'GCF_900130045.1_IMG-taxon_2622736499_annotated_assembly_protein.faa', 'tax_scope': 'auto'}, {'email': '***@live.unc.edu', 'jobname': 'MM_czetdef0', 'numseqs': '1787', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '00:58:54', 'original_file': 'GCF_001735715.1_Formosa_sp._Hel3_A1_48_protein.faa', 'tax_scope': 'auto'}, {'email': '***@edu.ulisboa.pt', 'jobname': 'MM_z9wd7xgk', 'numseqs': '19152', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '01:34:17', 'original_file': 'CKAH01.proteins.fasta', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_325gz5lg', 'numseqs': '7083', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:24:30', 'original_file': 'Exp1_S6_MEGAHIT.genes.fasta', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_54lhnmg6', 'numseqs': '4464', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:19:03', 'original_file': 'E2_S7.genes (copy).fna', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_6lu1lv9s', 'numseqs': '5718', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:17:09', 'original_file': 'E2_S12.genes (copy).fna', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_cjs6yyy8', 'numseqs': '6901', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:44:09', 'original_file': 'Exp1_S1_MEGAHIT.genes.fasta', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_loupoixq', 'numseqs': '7961', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:30:03', 'original_file': 'E2_S9.genes (copy).fna', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_ssadiqqf', 'numseqs': '4775', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:11:55', 'original_file': 'E2_S8.genes (copy).fna', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_tdvoan5q', 'numseqs': '6092', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:28:10', 'original_file': 'Exp1_S5_MEGAHIT.genes.fasta', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_tswf6qrk', 'numseqs': '6965', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:35:40', 'original_file': 'Exp1_S2_MEGAHIT.genes.fasta', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_wrsrt0co', 'numseqs': '4911', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:11:03', 'original_file': 'E2_S10.genes (copy).fna', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_z82kn4ca', 'numseqs': '6244', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:25:34', 'original_file': 'E2_S11.genes (copy).fna', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_44521l68', 'numseqs': '6965', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:39:50', 'original_file': 'Exp1_S2_MEGAHIT.genes.fasta', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_cv8emnjr', 'numseqs': '4394', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:10:29', 'original_file': 'Exp1_S4_MEGAHIT.genes.fasta', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_k__ka8pd', 'numseqs': '6283', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:29:50', 'original_file': 'Exp1_S3_MEGAHIT.genes.fasta', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_u2qeal5x', 'numseqs': '4101', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:15:27', 'original_file': 'S7_SP.genes.fasta', 'tax_scope': 'auto'}, {'email': '***@ucsd.edu', 'jobname': 'MM_664fgo4q', 'numseqs': '12', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-11', 'elapsed': '00:44:30', 'original_file': 'oma.flyemeta.r4m1.contiginfo.prophage.fasta', 'tax_scope': 'auto'}, {'email': '***@ucsd.edu', 'jobname': 'MM_7qh7udv7', 'numseqs': '29', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-11', 'elapsed': '00:51:36', 'original_file': 'kefir_rad004.flyemeta.r4m1.contiginfo.prophage.fasta', 'tax_scope': 'auto'}, {'email': '***@ucsd.edu', 'jobname': 'MM_hiyjnnwh', 'numseqs': '26', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-11', 'elapsed': '00:48:41', 'original_file': 'kefir_rbk001.flyemeta.r4m1.contiginfo.prophage.fasta', 'tax_scope': 'auto'}, {'email': '***@ucsd.edu', 'jobname': 'MM_pip08n32', 'numseqs': '14', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-11', 'elapsed': '01:02:04', 'original_file': 'harbison.flyemeta.r4m1.contiginfo.prophage.fasta', 'tax_scope': 'auto'}, {'email': '***@ucsd.edu', 'jobname': 'MM_vsbasxbg', 'numseqs': '9', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-11', 'elapsed': '00:44:34', 'original_file': 'bayley.flyemeta.r4m1.contiginfo.prophage.fasta', 'tax_scope': 'auto'}, {'email': '***@ucsd.edu', 'jobname': 'MM_xfjut6bz', 'numseqs': '4', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-11', 'elapsed': '00:42:09', 'original_file': 'harbison.flyemeta.r4m1.contiginfo.phage.fasta', 'tax_scope': '10239'}, {'email': '***@tamu.edu', 'jobname': 'MM_cxms4jgc', 'numseqs': '2', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '00:01:11', 'original_file': 'a.txt', 'tax_scope': 'auto'}, {'email': '***@tamu.edu', 'jobname': 'MM_m1jexy3h', 'numseqs': '2', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '00:02:44', 'original_file': 'a.txt', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_34o52c2f', 'numseqs': '4595', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:05:39', 'original_file': 'GCF_002269565.1_ASM226956v1_translated  _cds.faa', 'tax_scope': '1236'}, {'email': '***@gmail.com', 'jobname': 'MM_onbmqw3a', 'numseqs': '1901', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:06:42', 'original_file': 'rast_protein_seq.faa', 'tax_scope': 'auto'}, {'email': '***@163.com', 'jobname': 'MM_sxubqjr2', 'numseqs': '21847', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '02:49:38', 'original_file': 'jw_pep.fasta', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_3lxzes6g', 'numseqs': '5954', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:07:24', 'original_file': '7b_protein sequance.fasta', 'tax_scope': '267888'}, {'email': '***@gmail.com', 'jobname': 'MM_i4nw3id7', 'numseqs': '5954', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:16:52', 'original_file': '7b_protein sequance.fasta', 'tax_scope': '1236'}, {'email': '***@gmail.com', 'jobname': 'MM_aop41crs', 'numseqs': '2462', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:54:23', 'original_file': '683.faa', 'tax_scope': 'auto'}, {'email': '***@163.com', 'jobname': 'MM_15m8grrl', 'numseqs': '1278', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '00:44:25', 'original_file': 'output37.pep.fa', 'tax_scope': 'auto'}, {'email': '***@163.com', 'jobname': 'MM___2zvng_', 'numseqs': '670', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '00:45:19', 'original_file': 'output153.pep.fa', 'tax_scope': 'auto'}, {'email': '***@163.com', 'jobname': 'MM_b_6rvf_b', 'numseqs': '1477', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '00:48:33', 'original_file': 'output70.pep.fa', 'tax_scope': 'auto'}, {'email': '***@163.com', 'jobname': 'MM_dwn6h5wu', 'numseqs': '687', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '00:47:02', 'original_file': 'output128.pep.fa', 'tax_scope': 'auto'}, {'email': '***@163.com', 'jobname': 'MM_o996_8wq', 'numseqs': '837', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '00:51:28', 'original_file': 'output104.pep.fa', 'tax_scope': 'auto'}, {'email': '***@163.com', 'jobname': 'MM_r5h8cade', 'numseqs': '1115', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '00:44:05', 'original_file': 'output35.pep.fa', 'tax_scope': 'auto'}, {'email': '***@kiom.re.kr', 'jobname': 'MM_a2ebfvuc', 'numseqs': '7029', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '01:05:38', 'original_file': 'M.faa', 'tax_scope': 'auto'}, {'email': '***@hotmail.com', 'jobname': 'MM_9jxi91ug', 'numseqs': '2651', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '01:02:21', 'original_file': 'st5.faa', 'tax_scope': 'auto'}, {'email': '***@foxmail.com', 'jobname': 'MM_ap4kqqb9', 'numseqs': '4175', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '01:06:39', 'original_file': 'GCF_000746645.1_ASM74664v1_cds_from_genomic.fna', 'tax_scope': 'auto'}, {'email': '***@zjou.edu.cn', 'jobname': 'MM_bnr5bxgq', 'numseqs': '21847', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '02:41:22', 'original_file': 'jw_pep.fasta', 'tax_scope': '7742'}, {'email': '***@gatech.edu', 'jobname': 'MM_1he2mhz5', 'numseqs': '4800', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:33:16', 'original_file': 'CGT1079_trim_assembled_protein.faa', 'tax_scope': '2'}, {'email': '***@gatech.edu', 'jobname': 'MM_71vqj3xo', 'numseqs': '4711', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:26:05', 'original_file': 'CGT1096_trim_assembled_protein.faa', 'tax_scope': '2'}, {'email': '***@gatech.edu', 'jobname': 'MM_ojd5exen', 'numseqs': '4537', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:17:08', 'original_file': 'CGT1001_trim_assembled_protein.faa', 'tax_scope': '2'}, {'email': '***@unsw.edu.au', 'jobname': 'MM_9_0eew8f', 'numseqs': '1023', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:48:33', 'original_file': 'AD1_24_VS_AD10_24_DE_RUVg_rhodophyta.fasta', 'tax_scope': 'auto'}, {'email': '***@uwm.edu', 'jobname': 'MM_z008gju5', 'numseqs': '19828', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:19:07', 'original_file': 'PROKKA_03132020.faa', 'tax_scope': '1117'}, {'email': '***@gmail.com', 'jobname': 'MM_j51_xymq', 'numseqs': '34', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '00:03:10', 'original_file': 'phage776.faa', 'tax_scope': '10239'}, {'email': '***@uzh.ch', 'jobname': 'MM_ecw9jit4', 'numseqs': '52089', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '03:08:44', 'original_file': 'DR.faa', 'tax_scope': 'auto'}, {'email': '***@etu.u-bordeaux.fr', 'jobname': 'MM_13v70c3e', 'numseqs': '2563', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '01:11:55', 'original_file': 'lactis-CDSs.fasta', 'tax_scope': 'auto'}, {'email': '***@etu.u-bordeaux.fr', 'jobname': 'MM_pi_aaot5', 'numseqs': '3112', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '01:02:32', 'original_file': 'plantarum-CDSs.fasta', 'tax_scope': 'auto'}, {'email': '***@etu.u-bordeaux.fr', 'jobname': 'MM_rl953ntq', 'numseqs': '2593', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '01:00:57', 'original_file': 'freudenreichii-CDSs.fasta', 'tax_scope': 'auto'}, {'email': '***@hud.ac.uk', 'jobname': 'MM_6o65wi1_', 'numseqs': '416', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '00:44:32', 'original_file': 'LmBottom5.txt', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_rw4tc3k3', 'numseqs': '11673', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:31:50', 'original_file': 'polished_b07_Protein.faa', 'tax_scope': 'auto'}, {'email': '***@outlook.com', 'jobname': 'MM_2qqmoxul', 'numseqs': '639', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:44:32', 'original_file': 'fsk_paralogs.fasta', 'tax_scope': 'auto'}, {'email': '***@outlook.com', 'jobname': 'MM_2rjw5zgb', 'numseqs': '5509', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '00:49:30', 'original_file': 'unassigned.fasta', 'tax_scope': 'auto'}, {'email': '***@outlook.com', 'jobname': 'MM_c48_ztk4', 'numseqs': '3617', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:45:45', 'original_file': 'fsk_unassigned.fasta', 'tax_scope': 'auto'}, {'email': '***@outlook.com', 'jobname': 'MM_pi5exhrv', 'numseqs': '1278', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '00:46:08', 'original_file': 'paralogs.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_1j7hbvi1', 'numseqs': '36', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:02:18', 'original_file': 'GCA_001982335_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_62ccui1g', 'numseqs': '887', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:52:39', 'original_file': 'P1_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_9kqfp79d', 'numseqs': '47', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:02:35', 'original_file': 'GCA_001982385_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_9pdrkt9i', 'numseqs': '392', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:47:02', 'original_file': 'GCA_001982345_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_b3x9hp5q', 'numseqs': '22', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:02:05', 'original_file': 'GCA_001982185_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_cg7h35ln', 'numseqs': '164', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:03:40', 'original_file': 'GCA_001982195_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_edshe_5o', 'numseqs': '365', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:55:18', 'original_file': 'Lb1_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_f56dyjb6', 'numseqs': '144', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:03:29', 'original_file': 'GCA_00192115_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_ga4_g7g5', 'numseqs': '205', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:08:10', 'original_file': 'GCA_001982405_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_gl3041dd', 'numseqs': '680', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:49:34', 'original_file': 'GCA_001368775_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_l5uuzp_c', 'numseqs': '25', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:01:51', 'original_file': 'GCA_001982205_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_oqifgtr0', 'numseqs': '290', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:47:39', 'original_file': 'GCA_001982265_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_puxfq5_h', 'numseqs': '320', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:50:13', 'original_file': 'GCA_001368755_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_q31v5zhp', 'numseqs': '40', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:02:53', 'original_file': 'GCA_001982165_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_ukswrbcw', 'numseqs': '65', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:03:00', 'original_file': 'GCA_001982245_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@usp.br', 'jobname': 'MM_vj37ie09', 'numseqs': '1144', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:50:35', 'original_file': 'Lb2_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@univ-amu.fr', 'jobname': 'MM_827uuxjt', 'numseqs': '3', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:01:24', 'original_file': 'oscarella.fasta', 'tax_scope': 'auto'}, {'email': '***@univ-amu.fr', 'jobname': 'MM__83l94bm', 'numseqs': '11', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:01:59', 'original_file': 'bd.fasta', 'tax_scope': 'auto'}, {'email': '***@univ-amu.fr', 'jobname': 'MM_it2ykk35', 'numseqs': '11', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:02:00', 'original_file': 'bd.fasta', 'tax_scope': '33208'}, {'email': '***@163.com', 'jobname': 'MM_4kdur6cb', 'numseqs': '1', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:01:11', 'original_file': '新建文本文档.fasta', 'tax_scope': 'auto'}, {'email': '***@163.com', 'jobname': 'MM_krppospy', 'numseqs': '1', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:00:52', 'original_file': '新建文本文档.fasta', 'tax_scope': 'auto'}, {'email': '***@yonsei.ac.kr', 'jobname': 'MM_4_lrj674', 'numseqs': '1593', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '00:48:08', 'original_file': 'annot_protein_fasta.txt', 'tax_scope': '33958'}, {'email': '***@yonsei.ac.kr', 'jobname': 'MM_asb23vqn', 'numseqs': '1190', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '00:46:01', 'original_file': 'annot_protein_fasta.txt', 'tax_scope': '33958'}, {'email': '***@yonsei.ac.kr', 'jobname': 'MM_i0872ry_', 'numseqs': '2146', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '00:50:07', 'original_file': 'annot_protein_fasta.txt', 'tax_scope': '81850'}, {'email': '***@shanghaitech.edu.cn', 'jobname': 'MM_86gwym4r', 'numseqs': '6561', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '01:30:21', 'original_file': 'WM276_protein.faa', 'tax_scope': 'auto'}, {'email': '***@shanghaitech.edu.cn', 'jobname': 'MM_l5gmg080', 'numseqs': '7826', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '01:11:22', 'original_file': 'H99_protein.faa', 'tax_scope': 'auto'}, {'email': '***@shanghaitech.edu.cn', 'jobname': 'MM_mvmvvbnf', 'numseqs': '6472', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-14', 'elapsed': '01:10:37', 'original_file': 'R265_protein.faa', 'tax_scope': 'auto'}, {'email': '***@gmail.com', 'jobname': 'MM_97j3eh6x', 'numseqs': '4883', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:23:06', 'original_file': 'Dada-proteins.missing.eggNOG-sub.faa', 'tax_scope': 'auto'}, {'email': '***@unil.ch', 'jobname': 'MM_4s5dtt18', 'numseqs': '16', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:01:52', 'original_file': 'CP046135.faa', 'tax_scope': 'auto'}, {'email': '***@unil.ch', 'jobname': 'MM_gljsxik5', 'numseqs': '7', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:01:08', 'original_file': 'CP046133.faa', 'tax_scope': 'auto'}, {'email': '***@unil.ch', 'jobname': 'MM_hfxhebc6', 'numseqs': '6', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:01:24', 'original_file': 'CP046132.faa', 'tax_scope': 'auto'}, {'email': '***@yahoo.com.br', 'jobname': 'MM_4fqe0xu6', 'numseqs': '1920', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '00:49:44', 'original_file': '1920.fasta', 'tax_scope': 'auto'}, {'email': '***@yahoo.com.br', 'jobname': 'MM__9odvofr', 'numseqs': '35096', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:32:30', 'original_file': 'todas.fasta', 'tax_scope': 'auto'}, {'email': '***@yahoo.com.br', 'jobname': 'MM_ged7sagr', 'numseqs': '1127', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-12', 'elapsed': '00:47:55', 'original_file': '1920_x_NR.fasta', 'tax_scope': 'auto'}, {'email': '***@york.ac.uk', 'jobname': 'MM_a3baxr8o', 'numseqs': '105', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:03:55', 'original_file': 'PseudoPlasmid.feature_protein.fasta', 'tax_scope': 'auto'}, {'email': '***@163.com', 'jobname': 'MM_z88y964m', 'numseqs': '4228', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '01:06:20', 'original_file': 'sample1.pep', 'tax_scope': 'auto'}, {'email': '***@tugraz.at', 'jobname': 'MM_t9smdw3c', 'numseqs': '16', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:01:42', 'original_file': 'eggnog.fasta', 'tax_scope': '2'}, {'email': '***@tugraz.at', 'jobname': 'MM_yp45mf6o', 'numseqs': '2', 'databases': 'none', 'status': 'Done', 'submitted': '2020-03-13', 'elapsed': '00:01:12', 'original_file': 'eggnog_483219.LILAB_30110.fasta', 'tax_scope': 'auto'}];
    var csrf_token = 'UCKEl404brIzBSvM6wv5j8DQ5FRIo3RFWKPcTTzNvikE7yAk04MeWGmJar8PR0E6';
  </script>
  
  
  <script type="module" src="static/metronic/server_status.js"></script>
  <script type="module" src="static/metronic/eggnog_mapper.js"></script>
</html>
